docs/api-docs/customers/passwordless-login.md

```diff
@@ -51,9 +52,9 @@ Example:
 
 Upon receiving a successful `POST` request, BigCommerce will send a response that contains:
 
-* `expiry`: The time in seconds during which the login link is valid
-* `sent_email`: A value of `sign_in` indicates BigCommerce sent the login link to the customer via the email provided. A value of `password_reset` means BigCommerce emailed the customer a link with password reset instructions because they were previously flagged as needing to reset their password.
-
+* `expiry`: The time in seconds during which the login link is valid.
+* `sent_email`: A value of `sign_in` indicates BigCommerce sent the login link to the customer via the email provided. A value of `password_reset` means BigCommerce emailed the customer a link with password reset instructions because the customer requested a new password.
```

Comment:
This changes the meaning the value `password_reset`. 
* Does it mean the customer needs to reset their password, but hasn't yet? If so, the text was correct as is. 
* Or does it mean the customer requested a password reset? If so, your change might be more accurate.

Reply:
Luis Sanchez said said the following:
It means the customer requested a sign in link, but because it's been flagged as "reset password on next login", they will receive instead a reset password email

Reply:
The changes have been added.